backend-tests/tests/infra/cli.py

```diff
@@ -18,8 +18,19 @@ class CliUseradm:
     def __init__(self):
         self.cid = docker.getid('mender-useradm')
 
+        # is it an open useradm, or useradm-enterprise?
+        for path in ['/usr/bin/useradm', '/usr/bin/useradm-enterprise']:
```

Comment:
I'm comfortably removed from most of the complications here, so maybe you have good reasons, but I want to point out: Is it a good idea to use different binary names? My thinking is that if useradm-enterprise is going to be a drop in replacement for useradm, then it makes sense to keep the same name. This might help us later in tests that test both of them, for example. Just a gut feeling. :-)

Reply:
IMO, it's a different service so the binary name should be different;
I know useradm-enterprise is a replacement for useradm (same as for other enterprise services except for tenantadm), but to me, it's better to know exactly which service I'm using (open or enterprise) and the binary name is a good differentiator
I'm also aware that this will complicate tests and documentation so I don't have really strong opinion here...

Reply:
both points are valid and it could be decided either way. I'd keep the -enterprise distinction for now, mostly for the purpose of time - I feel we should proceed quickly forward now.

I do admit that at some point I may do a quick wholesale refactor of both useradm-enterprise and deployments-enterprise. Especially when this gives us problems in other areas, like docs.